capstone/backend/src/test/java/com/google/GetUserListsServletTest.Java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.BufferedReader;
+import java.io.StringWriter;
+import java.io.StringReader;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import junit.framework.TestCase;
+import org.mockito.Mockito;
+
+public class GetUserListsServletTest extends TestCase {
+  private static final String CORRECT_REQUEST_STRING = "{\"userId\": 1}";
+  private static final String INCORRECT_REQUEST_STRING = "{\"bad response\" list:\"bread\"}";
+
+  private class SetupObj {
+    HttpServletRequest request;
+    HttpServletResponse response;
+    StringWriter writer;
+
+    public SetupObj(HttpServletRequest request, HttpServletResponse response, 
+        StringWriter writer) {
+      this.request = request;
+      this.response = response;
+      this.writer = writer;
+    }
+  }
+
+  public void testDoPostSucceed() throws ServletException, IOException {
+    SetupObj setupObj = setupMockData(CORRECT_REQUEST_STRING);
+
+    CreateOrUpdateUserlistServlet servlet = new GetUserListsServlet();
+    servlet.doPost(setupObj.request, setupObj.response);
+
+    Mockito.verify(setupObj.response).setStatus(HttpServletResponse.SC_OK);
+    String result = setupObj.writer.toString();
+    assertTrue("should be valid json format", 
```

Comment:
You don't need to check the validness of the string here. If you really want, please use a more complete solution as described here: https://stackoverflow.com/questions/10174898/how-to-check-whether-a-given-string-is-valid-json-in-java.